src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,343 @@
+    }
+
+    @Override
+    public void closeTool() {
+        for (Map.Entry<String, FeatureReader<VariantContext>> reader : sampleToVCFMap.entrySet()) {
+            try {
+               reader.getValue().close();
+            } catch (IOException e) {
+                throw new GATKException("FeatureReader close() for " + reader.getKey() +
+                    " has thrown an IOException: " + e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Before traversal starts, create the feature readers
+     * for all the input GVCFs, create the merged header and
+     * initialize the interval
+     */
+    @Override
+    public void onStartup() {
+
+        for (String variantPath : variantFilePaths) {
+
+            final String variantURI = IOUtils.getPath(variantPath).toAbsolutePath().toUri().toString();
+            final Function<SeekableByteChannel, SeekableByteChannel> cloudWrapper = (cloudPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudPrefetchBuffer, is) : Function.identity());
+            final Function<SeekableByteChannel, SeekableByteChannel> cloudIndexWrapper = (cloudIndexPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudIndexPrefetchBuffer, is) : Function.identity());
+            AbstractFeatureReader<VariantContext, LineIterator> reader = AbstractFeatureReader.getFeatureReader(variantURI, null, new VCFCodec(), true, cloudWrapper, cloudIndexWrapper);
+
+            // A GVCF file must contain only one sample, throw an exception otherwise
+            if (((VCFHeader)reader.getHeader()).getGenotypeSamples().size() > 1) {
+                throw new UserException("Input GVCF: " + variantPath + " contains more than one sample");
+            }
+
+            String sampleName = ((VCFHeader) reader.getHeader()).getGenotypeSamples().get(0);
+            sampleToVCFMap.put(sampleName, reader);
+        }
+
+        createMergedHeader();
+
+        initializeIntervals();
+
+        super.onStartup();
+    }
+
+    /**
+     * Create merged header from input GVCFs
+     */
+    private void createMergedHeader() {
+        List<VCFHeader> headers = new ArrayList<>();
+        for (Map.Entry<String, FeatureReader<VariantContext>> sampleToVCF : sampleToVCFMap.entrySet()) {
+            FeatureReader<VariantContext> reader = sampleToVCF.getValue();
+            VCFHeader vcfHeader = (VCFHeader)reader.getHeader();
+            headers.add((VCFHeader)reader.getHeader());
+        }
+
+        mergedHeader = VCFUtils.smartMergeHeaders(headers, true);
+        mergedVCFHeader = new VCFHeader(mergedHeader);
+    }
+
+    /**
+     * Loads our intervals using the best available sequence
+     * dictionary (as returned by {@link #getBestAvailableSequenceDictionary})
+     * to parse/verify them. Does nothing if no intervals were specified.
+     */
+    private void initializeIntervals() {
+        if (intervalArgumentCollection.intervalsSpecified()) {
```

Comment:
Throw a `UserException` if `intervalArgumentCollection.intervalsSpecified()` is false